tabular/src/autogluon/tabular/models/tabular_nn/hyperparameters/searchspaces.py

```diff
@@ -61,3 +63,16 @@ def get_searchspace_regression():
         # 'batch_size': Categorical(512, 1024, 2056, 128), # this is used in preprocessing so cannot search atm
     }
     return params
+
+
+def get_searchspace_quantile():
+    params = {
+        'learning_rate': Categorical(1e-4, 5e-4, 1e-3),
+        'weight_decay': Categorical(1e-7, 1e-6, 1e-5),
+        'dropout_prob': Categorical(0.0, 0.1, 0.2),
+        'gamma': Real(0.1, 10.0, default=5.0),
+        'num_layers': Categorical(2, 3, 4),
+        'hidden_size': Categorical(128, 256, 512),
+        'embedding_size_factor': Real(0.5, 1.5, default=1.0),
```

Comment:
Did you have a good reason for deviating from the default search space for QUANTILE problem_type?

Reply:
The first thing is, QUANILTE setting is using pytorch instead of mxnet.
The second is, this search space is the one I used in our DQA experiments.